scripts/release/prepublish_check.py

```
@@ -0,0 +1,321 @@
+                    f.fail(f"{name}: {key} points at '{v}' which is not in the tarball")
+
+        # Any header that #includes a sibling must find it. This is exactly how
+        # @runanywhere/core 0.20.18 shipped unbuildable for RN Android.
+        hdrs = {r for r in rel if r.endswith(".h")}
+        if hdrs:
+            missing: set[str] = set()
+            for h in hdrs:
+                try:
+                    body = tf.extractfile("package/" + h).read().decode("utf-8", "ignore")
+                except Exception:
+                    continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)